apps/src/sites/studio/pages/home/_homepage.js

```diff
@@ -165,3 +166,39 @@ function showHomepage() {
     document.getElementById('homepage-container')
   );
 }
+
+window.CleverTakeoverManager = function (options) {
+  this.options = options;
```

Comment:
Nit: You're closed over the `options` argument here, so you can omit this assignment and just refer to `options` any place you are using `this.options` or `self.options`.  You won't have to think about `this` binding at all.

It might be even more expressive to do a destructing assignment right in the function parameter list, which would make explicit the available options and let the rest of your code get even more compact.

```js
window.CleverTakeoverManager = function ({
  cleverLinkFlag,
  userIDToMerge,
  mergeAuthToken
}) {
  // ...rest of the module...

  // Inside onConfirmLink:
  window.location.href = `/users/clever_takeover?mergeID=${userIDToMerge}&token=${mergeAuthToken}`;
};
```